### ISSUE TITLE
tf.bfloat16 is unsigned on Windows

### ISSUE THREAD

**Issue by snnn** [tensorflow/tensorflow#15297]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Windows 10
- **TensorFlow installed from (source or binary)**:
binary
- **TensorFlow version (use command below)**:
1.4.0
- **Python version**: 
3.5.4
- **Bazel version (if compiling from source)**:
None
- **GCC/Compiler version (if compiling from source)**:
None
- **CUDA/cuDNN version**:
None
- **GPU model and memory**:
None
- **Exact command to reproduce**:
```python
import tensorflow as tf
print(tf.bfloat16.is_unsigned)
```
Expected Output: False
Actual: True

### Source code / logs
https://ci.tensorflow.org/job/tf-master-win-bzl/2057/console
bfloat16_test/test.log

**Comment by gunan** [comment]
Did we recently introduce this breakage, or has this always been in our code?

**Comment by snnn** [comment]
It only affects TF 1.4 and the nightly build. TF 1.3 is ok. It was introduced in 390dcd29ac75ba59a3ff6ab45ab58f80d4094dc2

**Comment by snnn** [comment]
Hi @meteorcloudy  
Can you review that PR?

Thanks.

**Comment by snnn** [comment]
#15302 is merged, but the code is not compilable right now. Should we left this issue open or close it?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by meteorcloudy** [comment]
Sorry for the late reply, is #15466 a requirement for #15302?
I don't quite understand the solution, is there any other way to fix it?

**Comment by meteorcloudy** [comment]
Both of the tests are failing, so I'll have to reopen this issue.

**Comment by snnn** [comment]
Hi @meteorcloudy 

You are right.

**Comment by tensorflowbutler** [comment]
A member of the TensorFlow organization has replied after the stat:awaiting tensorflower label was applied.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by snnn** [comment]
Hi @meteorcloudy

What's the status of this issue?  Is it fixed or not?

**Comment by meteorcloudy** [comment]
Sorry, I was on vacation for the last two weeks. And I don't know what's the proper way to fix it.
@martinwicke Can you point this issue to the correct owner?

**Comment by tensorflowbutler** [comment]
Nagging Assignees @meteorcloudy, @gunan: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by martinwicke** [comment]
ping @snnn

**Comment by snnn** [comment]
Hi @martinwicke ,

I'm glad to take it.  Recently I'm quite busy, I have a lot of family stuffs to deal with. I just had a 3 months leave, for my new born baby. :-) I'll update this thread in the next week.

**Comment by martinwicke** [comment]
No worries! Congratulations!

**Comment by tensorflowbutler** [comment]
Nagging Assignees @meteorcloudy, @gunan: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by rongjiecomputer** [comment]
@snnn I fixed this with #18047.

```python
>>> import tensorflow as tf
>>> print(tf.bfloat16.is_unsigned)
False
```

**Comment by snnn** [comment]
Great! Thank you!
